frontend/src/store/controller/selectors/frontend_pb.ts

```diff
@@ -0,0 +1,60 @@
+import { createSelector } from 'reselect';
```

Comment:
The naming of this file is a bit awkward; but because in the future we'll want to move selectors out to feature folders (e.g. a feature folder for the rotary encoder behaviors, maybe a feature folder for system settings, etc.), the filename you chose is fine for this PR.